frontend/coprs_frontend/alembic/versions/d23f84f87130_eol_rolling_builds.py

```diff
@@ -0,0 +1,30 @@
+"""
+record last copr_chroot build, allow marking Rawhide as rolling
+Create Date: 2024-05-13 08:56:31.557843
+"""
+
+import time
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.sql import text
+
+revision = 'd23f84f87130'
+down_revision = '41763f7a5185'
+
+def upgrade():
+    op.add_column('mock_chroot', sa.Column('rolling', sa.Boolean(), nullable=True))
+    op.add_column('copr_chroot', sa.Column('last_build_timestamp', sa.Integer(), nullable=True))
+    conn = op.get_bind()
+    conn.execute(
+        text("update copr_chroot set last_build_timestamp = :start_stamp;"),
+        {"start_stamp": int(time.time())},
```

Comment:
note that we do not remove anything sooner than in 360 days till we run this migration